cypress/integration/02 - items.ts

```diff
@@ -0,0 +1,68 @@
+const testItem = {
+    name: "Google",
+    username: "example@google.com",
+    password: "somethingsecret",
+    url: "https://google.com",
+};
+
+describe("Items", () => {
+    it("can create an item without errors", () => {
+        cy.login();
+
+        // Click plus sign
+        cy.get("pl-app").find("pl-items").find("pl-items-list").find("pl-button:eq(2)").click({ force: true });
+
+        // Give the app some time to render the animations
+        cy.wait(100);
+
+        // Click create
+        cy.get("pl-app").find("pl-create-item-dialog").find("footer pl-button.primary").click({ force: true });
+
+        cy.url().should("include", "/items/");
+        cy.url().should("include", "/new");
+
+        // Give the app some (more) time to render the animations
+        cy.wait(100);
```

Comment:
This _might_ need to be increased a bit more (this animation is slower), if, once we add the "find item", we can't consistently get the item properly typed.